flow/flow.go

```diff
@@ -627,8 +659,8 @@ func (f *Flow) newTransportLayer(packet *gopacket.Packet) error {
 
 // PacketsFromGoPacket split original packet into multiple packets in
 // case of encapsulation like GRE, VXLAN, etc.
-func PacketsFromGoPacket(packet *gopacket.Packet, outerLength int64, t int64, bpf *BPF) *Packets {
-	flowPackets := &Packets{Timestamp: t}
+func PacketsFromGoPacket(packet *gopacket.Packet, outerLength int64, bpf *BPF) *Packets {
```

Comment:
It seems you broke the replay

Reply:
you can't remove the t argument it is used by the replay mechanism

Reply:
yeah I seen it's a big trick to force replay to works, so it's specific code for test only ...

Reply:
No this is not for test purpose it is to be able to reinject with the exact same timing in order to simulate more or less real traffic